### PR TITLE
current status for Instart Logic implementation as of 6/1/2016

### PR DESCRIPTION
**Pull request by wesleyhales**

### CODE REVIEW

**Comment by igrigorik** [comment]
@wesleyhales awesome! Do we have any links to docs or announcements for how to enable/configure, etc?

**Comment by wesleyhales** [comment]
This was verified on an internal thread, but I'm pretty sure it's all internal (non-customer facing) configurations.

File: index.html [hunk 1]
```diff
@@ -430,11 +430,11 @@ <h3 id="cdn-paas"><a href="#cdn-paas">CDN &amp; PaaS performance</a></h3>
             <td>Instart Logic</td>
             <td class="ok">yes</td>
             <td class="ok">yes</td>
-            <td class="alert">no</td>
+            <td class="ok">yes</td>
             <td class="warn">configurable (static)</td>
+            <td class="warn">no</td>
```
Comment (igrigorik) [inline index.html]: alert? :)

**Comment by igrigorik** [comment]
Hmm, ok. It would be great to have docs linking to the right places if you have any -- we can tackle that as a separate pull request though.

**Comment by wesleyhales** [comment]
Ok. If the public docs don't exist. I will lead the effort on creating them ;) I agree that it should be out there to warrant this PR

File: index.html (hunk 1 above)
Comment (wesleyhales) [inline index.html]: Funny

Comment (sc0ttbeardsley) [inline index.html]: This column is for ALPN. it should be "yes":
```
$ openssl s_client -connect www.instartlogic.com:443 -alpn h2 </dev/null 2>/dev/null|grep ALPN
...
ALPN protocol: h2
```

Also H2 is supported and was announced here: https://www.instartlogic.com/blog/upgrading-http2

@wesleyhales can you add that link to the last column?

**Comment by sc0ttbeardsley** [comment]
👍

**Comment by igrigorik** [comment]
Thanks guys! Live.

**Pull request merged**